README.md

```diff
@@ -0,0 +1,568 @@
+# PhD - PHP DocBook
+
+## About PhD
+
+PhD is PHP's very own DocBook 5 rendering system. It is used to convert
+the PHP Manual and PEAR Documentation into different output formats like
+XHTML, PDF, Man pages and CHM.
+
+The goal of PhD is to become a fast, general DocBook rendering system.
+At the moment of writing, PhD is already very fast: It can create the
+chunked version of PEAR's manual (some 3000 files) in less than a minute
+on a 2GHz system. It also renders the PHP and PEAR manual flawlessly. It
+does not support every DocBook 5 tag yet, and using it to render your own
+DocBook files may need some tweaks.
+
+## Requirements
+
+Requirements:
+- PHP 8.0+
+- DOM, libXML2, XMLReader and SQLite3.
+
+## Quickstart
+
+Get PhD from the source repository
+
+```shell
+$ git clone https://github.com/php/phd.git
+```
+
+and render your documentation file (`doc-base/.manual.xml` in this example)
+in `php` format
+
+```shell
+$ php phd/render.php --docbook doc-base/.manual.xml --package PHP --format php
+```
+
+
+## Installation
+
+PhD is distributed via its own PEAR channel, [doc.php.net](http://doc.php.net).
+Using this channel is also the easiest way to get it.
+
+### Installation via PEAR
+
+> **_NOTE:_**
+> You need a working [PEAR
+> installation](https://pear.php.net/manual/en/installation.php).
+
+To install the latest version of PhD:
+
+```shell
+$ pear install doc.php.net/phd
+```
+
+Verify the installation:
+```shell
+$ phd --version
+PhD Version: 1.0.0-stable
+PHP Version: 5.3.3
+Copyright(c) 2007-2024 The PHP Documentation Group
+```
+
+To see the list of all available PhD packages:
+
+```shell
+$ pear remote-list -c doc.php.net
+```
+
+Installing the PhD Packages:
+
+```shell
+$ pear install doc.php.net/phd_php
+Starting to download PhD_PHP-1.0.0.tgz (18,948 bytes)
+[...]
+install ok: channel://doc.php.net/PhD_PHP-1.0.0
+
+$ pear install doc.php.net/phd_pear
+downloading PhD_PEAR-1.0.0.tgz ...
+[...]
+install ok: channel://doc.php.net/PhD_PEAR-1.0.0
+```
+
+### Installation from Git
+
+To get the latest and greatest features that have not been released yet,
+you can use PhD from Git.
+
+```shell
+$ git clone https://github.com/php/phd.git
+... output
+
+$ pear install package.xml package_generic.xml package_php.xml package_pear.xml
+[...]
+install ok: channel://doc.php.net/PhD-1.0.1
+install ok: channel://doc.php.net/PhD_Generic-1.0.1
+install ok: channel://doc.php.net/PhD_PHP-1.0.1
+install ok: channel://doc.php.net/PhD_PEAR-1.0.1
+```
+
+To install the standalone Packages:
+```shell
+for i in package_*.xml; do pear install $i; done
+```
+
+```shell
+$ phd --version
+PhD Version: phd-from-svn
+PHP Version: 5.3.3-dev
+Copyright(c) 2007-2010 The PHP Documentation Group
+```
```

Comment:
Actually this one as well, as I don't even know if "installation" works, and really should not be done